crates/ouroboros-traits/src/is_header/mod.rs

```diff
@@ -0,0 +1,76 @@
+// Copyright 2025 PRAGMA
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use amaru_kernel::{cbor, Hash, Hasher, Header, Point};
+
+pub mod fake;
+
+pub const HASH_SIZE: usize = 32;
+
+/// Interface to a header for the purpose of chain selection.
+pub trait IsHeader: cbor::Encode<()> + for<'d> cbor::Decode<'d, ()> {
+    /// Hash of the header
+    ///
+    /// This is used to identify the header in the chain selection.
+    /// Header hash is expected to be unique for each header, eg.
```

Comment:
Is this comment really necessary? A cryptographic hash is expected to uniquely identify a piece of data anyway, headers are not special in that respect.

Reply:
I don't know, you wrote it 🙈  ( https://github.com/pragma-org/amaru/commit/91f7905fe70c226323b933b45fc436acad844170#diff-ae8f32ce58b881f1f7c46be7c48fc53d4e7e79ee308c4d8a451b4437fd2f7f76R5-R10 )

Felt rude to remove it.